wp-content/themes/pub/wporg-learn-2024/src/style/style.scss

```diff
@@ -39,3 +39,28 @@ body {
 		padding-top: unset !important;
 	}
 }
+
+.wporg-learn-sidebar-meta-info {
+	.wp-block-sensei-lms-button-take-course {
+		margin-top: 0;
+		margin-bottom: var(--wp--preset--spacing--20);
+
+		> button {
+			line-height: 1;
+		}
+	}
+
+	.wporg-learn-sidebar-playground  a {
+		line-height: 1;
```

Comment:
Not possible with block markup?

Reply:
It does have line-height but it only applies to the parent div. Same for the [decoration](https://github.com/WordPress/Learn/blob/fe056e2de03365a076c31976b5651fa3fce0b045/wp-content/themes/pub/wporg-learn-2024/src/style/style.scss#L57-L60).

<img width="3338" alt="image" src="https://github.com/WordPress/Learn/assets/18050944/dc417896-36d6-4a7f-b1bf-fd8a0fd66e1f">

And [it](https://github.com/WordPress/Learn/blob/2388-align-single-course-with-design/wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-meta-info.php#L39) doesn't have a markup.